# HG changeset patch
# User Avlas <jsardid@gmail.com>
# Date 1523643442 25200
#      Fri Apr 13 11:17:22 2018 -0700
# Branch stable
# Node ID e5341aff3cb8839b41a4beff66cd8fbd9d7c2217
# Parent  e74d85f7ac47c026bb5ed9ae0c25cc2d3b1acd3b
Update Catalan translation (bug #53006).

* libgui/languages/ca_ES.ts: Update translation.

diff --git a/libgui/languages/ca_ES.ts b/libgui/languages/ca_ES.ts
--- a/libgui/languages/ca_ES.ts
+++ b/libgui/languages/ca_ES.ts
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='utf-8'?>
 <!DOCTYPE TS>
-<TS language="ca" version="2.1">
+<TS version="2.1" language="ca">
 <context>
     <name>QObject</name>
     <message>
         <location filename="../src/workspace-model.cc" line="+78"/>
         <source>automatic</source>
         <translation>automàtic</translation>
     </message>
     <message>
@@ -36,17 +36,16 @@
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+65"/>
         <source>foreground</source>
         <translation>primer pla</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>background</source>
-        
         <translation>segon plan</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
         <translation>selecció</translation>
     </message>
     <message>
@@ -142,17 +141,16 @@
     <message>
         <location line="+4"/>
         <source>Autoscale</source>
         <translation>Autoescala</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;File</source>
-        
         <translation>&amp;Fitxer</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Save</source>
         <translation>&amp;Desa</translation>
     </message>
     <message>
@@ -270,17 +268,16 @@
     <message>
         <location line="+6"/>
         <source>Text</source>
         <translation>Text</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>String</source>
-        
         <translation>Cadena de text</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Font</source>
         <translation>Tipus de lletra</translation>
     </message>
     <message>
@@ -306,17 +303,16 @@
     <message>
         <location line="+428"/>
         <source>color</source>
         <translation>color</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>Horz alignment</source>
-        
         <translation>Alineació horitzontal</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>left</source>
         <translation>esquerra</translation>
     </message>
     <message>
@@ -328,17 +324,16 @@
     <message>
         <location line="-24"/>
         <source>right</source>
         <translation>dreta</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Vert alignment</source>
-        
         <translation>Alineació vertical</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>top</source>
         <translation>part superior</translation>
     </message>
     <message>
@@ -389,23 +384,21 @@
     <message>
         <location line="+13"/>
         <source>Box</source>
         <translation>Casella</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background</source>
-        
         <translation>Segon pla</translation>
     </message>
     <message>
         <location line="+444"/>
         <source>Edge</source>
-        
         <translation>Vora</translation>
     </message>
     <message>
         <location line="+133"/>
         <source>Line style</source>
         <translation>Estil de línia</translation>
     </message>
     <message>
@@ -632,17 +625,17 @@ en mode escriptura: %2.</translation>
     <message>
         <location line="+5"/>
         <source>&amp;Redo</source>
         <translation>&amp;Refés</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Cu&amp;t</source>
-        <translation>Talla&amp;r</translation>
+        <translation>Ta&amp;lla</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Find and Replace...</source>
         <translation>&amp;Cerca i reemplaça...</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -697,17 +690,16 @@ en mode escriptura: %2.</translation>
     <message>
         <location line="+7"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplica la selecció/línia</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Transpose Line</source>
-        
         <translation>Intercanvia la línia</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Show Completion List</source>
         <translation>&amp;Mostra la llista de compleció</translation>
     </message>
     <message>
@@ -738,23 +730,21 @@ en mode escriptura: %2.</translation>
     <message>
         <location line="+5"/>
         <source>Comment (Choosing String)</source>
         <translation>Comenta (Seleccionant la cadena de text)</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Indent Selection Rigidly</source>
-        
         <translation>&amp;Sagna estrictament la selecció</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Unindent Selection Rigidly</source>
-        
         <translation>&amp;Neteja estrictament el sagnat de la selecció</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Aplica el sagnat al codi</translation>
     </message>
     <message>
@@ -900,17 +890,17 @@ en mode escriptura: %2.</translation>
     <message>
         <location line="+6"/>
         <source>&amp;Next Breakpoint</source>
         <translation>Punt d'interrupció &amp;següent</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Pre&amp;vious Breakpoint</source>
-        <translation>Punto d'interrupció &amp;anterior</translation>
+        <translation>Punt d'interrupció &amp;anterior</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>&amp;Remove All Breakpoints</source>
         <translation>&amp;Elimina tots els punts d'interrupció</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -958,17 +948,16 @@ en mode escriptura: %2.</translation>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+140"/>
         <source>line:</source>
         <translation>Línia:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>col:</source>
-        
         <translation>Columna:</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>encoding:</source>
         <translation>Codificació:</translation>
     </message>
     <message>
@@ -1014,17 +1003,16 @@ Vols desar i afegir el punt d'interrupció o cancel·lar?</translation>
         <location line="+126"/>
         <source>Comment selected text</source>
         <translation>Comenta el text seleccionat</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Comment string to use:
 </source>
-        
         <translation>Cadena de text de comentari a utilitzar:</translation>
     </message>
     <message>
         <location line="+135"/>
         <location line="+60"/>
         <source>&lt;unnamed></source>
         <translation>&lt;sense nom></translation>
     </message>
@@ -1092,18 +1080,17 @@ is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
         <translation>"%1"
 no és un identificador vàlid.
 
-Si manteniu aquest nom de fitxer, no podreu
-cridar el vostre script amb el seu nom com a ordre d'Octave.
+Si manteniu aquest nom de fitxer, no podreu executar el codi pel seu nom a l'Octave.
 
 Vols seleccionar un altre nom?</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>The current encoding %1
 can not be applied.
 
@@ -1174,27 +1161,26 @@ ha estat esborrat o reanomenat. Vols des
     <message>
         <location line="+1"/>
         <source>Date modified</source>
         <translation>Data de modificació</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Show hidden</source>
-        <translation>Mostrar els ocults</translation>
+        <translation>Mostra els ocults</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternating row colors</source>
         <translation>Alternar els colors de les files</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>Enter the path or filename</source>
-        
         <translation>Introdueix el camí o el nom del fitxer</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>One directory up</source>
         <translation>Directori superior</translation>
     </message>
     <message>
@@ -1231,17 +1217,17 @@ ha estat esborrat o reanomenat. Vols des
         <location line="+12"/>
         <source>Set Browser Directory...</source>
         <translation>Estableix el directori del navegador...</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+365"/>
         <source>Find Files...</source>
-        <translation>Cerca arxius...</translation>
+        <translation>Cerca fitxers...</translation>
     </message>
     <message>
         <location line="-361"/>
         <location line="+374"/>
         <source>New File...</source>
         <translation>Nou fitxer...</translation>
     </message>
     <message>
@@ -1450,28 +1436,27 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+87"/>
         <source>Find and Replace</source>
         <translation>Cerca i reemplaça</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Find &amp;what:</source>
-        
         <translation>&amp;Cerca:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Re&amp;place with:</source>
         <translation>&amp;Reemplaça amb:</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Match &amp;case</source>
-        <translation>Distingir entre &amp;majúscules/minúscules</translation>
+        <translation>Distingeix entre &amp;majúscules/minúscules</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search from &amp;start</source>
         <translation>Cerca des de l'&amp;inici</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1559,27 +1544,27 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/find-files-dialog.cc" line="+54"/>
         <source>Find Files</source>
         <translation>Cerca fitxers</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Named:</source>
-        <translation>Anomenat:</translation>
+        <translation>Nom:</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Enter the filename search expression</source>
         <translation>Introdueix expressió de cerca per al nom de fitxer</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Start in:</source>
-        <translation>Comença en:</translation>
+        <translation>Comença a:</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Enter the start directory</source>
         <translation>Introdueix el directori inicial de cerca</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1609,43 +1594,42 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Include matching directories in search results</source>
         <translation>Inclou els directoris coincidents als resultats de cerca</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Name case insensitive</source>
-        <translation>No distingir entre majúscules i minúscules</translation>
+        <translation>No distingeixes entre majúscules i minúscules</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Set matching name is case insensitive</source>
         <translation>Estableix no distinció entre majúscules i minúscules</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Contains text:</source>
         <translation>Conté el text:</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Enter the file content search expression</source>
-        
         <translation>Introdueix l'expressió a cercar al contingut del fitxer</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Text to match</source>
         <translation>Text a coincidir</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Text case insensitive</source>
-        <translation>No distingir entre majúscules i minúscules</translation>
+        <translation>No distingeixes entre majúscules i minúscules</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Set text content is case insensitive</source>
         <translation>Estableix no distinció entre majúscules i minúscules</translation>
     </message>
     <message>
         <location line="+11"/>
@@ -1731,17 +1715,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Evaluate</source>
         <translation>Evalua</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Create script</source>
-        <translation>Crea un script</translation>
+        <translation>Bolca a un fitxer</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Hide filter</source>
         <translation>Amaga el filtre</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1926,27 +1910,27 @@ Feu clic a 'Següent' per crear un fitxer de configuració i iniciar l'Octave.&lt;/p>
     <message>
         <location line="+3"/>
         <source>Save Workspace As...</source>
         <translation>Desa l'espai de treball com a...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Exit</source>
-        <translation>Eixir</translation>
+        <translation>Sortir</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>New</source>
         <translation>Nou</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>New Script</source>
-        <translation>Nou script</translation>
+        <translation>Nou fitxer</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>New Function...</source>
         <translation>Nova funció...</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -2011,29 +1995,26 @@ Feu clic a 'Següent' per crear un fitxer de configuració i iniciar l'Octave.&lt;/p>
     <message>
         <location line="+49"/>
         <source>De&amp;bug</source>
         <translation>&amp;Depura</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step</source>
-        
         <translation>Pas (següent instrucció)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Step In</source>
-        
         <translation>Pas (entra)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Step Out</source>
-        
         <translation>Pas (surt)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Continue</source>
         <translation>Continua</translation>
     </message>
     <message>
@@ -2115,17 +2096,17 @@ Feu clic a 'Següent' per crear un fitxer de configuració i iniciar l'Octave.&lt;/p>
     <message>
         <location line="-36"/>
         <source>Variable Editor</source>
         <translation>Editor de variables</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Reset Default Window Layout</source>
-        <translation>Restaura el disseny de finestres per defecte</translation>
+        <translation>Restaura el disseny de les finestres per defecte</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
         <translation>&amp;Ajuda</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -2135,27 +2116,27 @@ Feu clic a 'Següent' per crear un fitxer de configuració i iniciar l'Octave.&lt;/p>
     <message>
         <location line="+3"/>
         <source>Octave Packages</source>
         <translation>Paquets de l'Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Contribute</source>
-        <translation>Contribuir</translation>
+        <translation>Contribueix</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Donate to Octave</source>
-        <translation>Donar a Octave</translation>
+        <translation>Donatius</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>On Disk</source>
-        <translation>En disc</translation>
+        <translation>Local</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Online</source>
         <translation>En línia</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -2331,29 +2312,26 @@ no existeix. Vols crear-lo?</translation
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Editor de l'Octave</translation>
     </message>
     <message>
         <location line="+252"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        
         <translation>El fitxer %1 no existeix als camins de càrrega. Per executar o depurar la funció que s'està editant s'ha de canviar al directori %2 o afegir aquest als camins de càrrega.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        
         <translation>Existeix un fitxer amb el mateix nom %1 al camí de càrrega. Per executar o depurar la funció que s'està editant desplaça't al directori %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        
         <translation>Canvia el directori o afegeix directori al camí de càrrega</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Change Directory</source>
         <translation>&amp;Canvia el directori</translation>
     </message>
     <message>
@@ -2554,29 +2532,26 @@ Vols sobreescriure-la suprimint l'acció anterior?</translation>
         <location line="+1"/>
         <location line="+162"/>
         <source>Preferences</source>
         <translation>Preferències</translation>
     </message>
     <message>
         <location line="-159"/>
         <source>Step</source>
-        
         <translation>Pas (següent instrucció)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Into</source>
-        
         <translation>Pas (entra)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Out</source>
-        
         <translation>Pas (surt)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Continue</source>
         <translation>Continua</translation>
     </message>
     <message>
@@ -2653,17 +2628,17 @@ Vols sobreescriure-la suprimint l'acció anterior?</translation>
     <message>
         <location line="+2"/>
         <source>Variable Editor</source>
         <translation>Editor de variables</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Reset Default Window Layout</source>
-        <translation>Restaura el disseny de finestres per defecte</translation>
+        <translation>Restaura el disseny de les finestres per defecte</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Ondisk Documentation</source>
         <translation>Mostra la documentació emmagatzemada</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2823,17 +2798,16 @@ Vols sobreescriure-la suprimint l'acció anterior?</translation>
     <message>
         <location line="+2"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplica la selecció/línia</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
-        
         <translation>Intercanvia la línia</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Completion List</source>
         <translation>Mostra la llista de compleció</translation>
     </message>
     <message>
@@ -3033,60 +3007,60 @@ Vols sobreescriure-la suprimint l'acció anterior?</translation>
         <location line="+57"/>
         <source>Global</source>
         <translation>Global</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+27"/>
         <source>File Menu</source>
-        <translation>Menú fitxer</translation>
+        <translation>Menú Fitxer</translation>
     </message>
     <message>
         <location line="-25"/>
         <location line="+27"/>
         <source>Edit Menu</source>
-        <translation>Menú edita</translation>
+        <translation>Menú Edita</translation>
     </message>
     <message>
         <location line="-25"/>
         <location line="+29"/>
         <source>Debug Menu</source>
-        <translation>Menú depuració</translation>
+        <translation>Menú Depura</translation>
     </message>
     <message>
         <location line="-27"/>
         <source>Window Menu</source>
-        <translation>Menú finestra</translation>
+        <translation>Menú Finestra</translation>
     </message>
     <message>
         <location line="+2"/>
         <location line="+29"/>
         <source>Help Menu</source>
-        <translation>Menú ajuda</translation>
+        <translation>Menú Ajuda</translation>
     </message>
     <message>
         <location line="-27"/>
         <source>News Menu</source>
-        <translation>Menú notícies</translation>
+        <translation>Menú Notícies</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Tab Handling in Dock Widgets</source>
         <translation>Comportament de les pestanyes en plafons acoblats</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>View Menu</source>
-        <translation>Menú visualitza</translation>
+        <translation>Menú Visualitza</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Run Menu</source>
-        <translation>Menú executa</translation>
+        <translation>Menú Executa</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Import shortcuts from file...</source>
         <translation>Importa dreceres des d'un fitxer...</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3265,17 +3239,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+5"/>
         <source>Plot selected data</source>
         <translation>Representa gràficament les dades seleccionades</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Up</source>
-        <translation>Cap amunt</translation>
+        <translation>Puja</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Go one level up in variable hierarchy</source>
         <translation>Puja un nivell cap amunt en la jerarquia de la variable</translation>
     </message>
 </context>
 <context>
@@ -3513,22 +3487,22 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="-186"/>
         <source>Octave logo only</source>
         <translation>Sols el logotip de l'Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
-        <translation>Icones de lletra</translation>
+        <translation>Icones tipogràfiques</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Graphic icons</source>
-        <translation>Icones de gràfic</translation>
+        <translation>Icones gràfiques</translation>
     </message>
     <message>
         <location line="-29"/>
         <source>Dock widget title bar</source>
         <translation>Acobla el plafó de la barra de títol</translation>
     </message>
     <message>
         <location line="-249"/>
@@ -3543,67 +3517,67 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+151"/>
         <source>Custom style</source>
         <translation>Estil personalitzat</translation>
     </message>
     <message>
         <location line="+170"/>
         <source>These preferences are applied after any .octaverc startup files.</source>
-        <translation>Aquestes configuracions s'apliquen després dels fitxers d'inici .octaverc.</translation>
+        <translation>Aquestes configuracions s'apliquen després del fitxer d'inici ".octaverc".</translation>
     </message>
     <message>
         <location line="+55"/>
         <source>Restore working directory of previous session</source>
-        <translation>Restaurar el directori de treball de la sessió anterior</translation>
+        <translation>Restaura el directori de treball de la sessió anterior</translation>
     </message>
     <message>
         <location line="+33"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
         <location line="+314"/>
         <source>Color</source>
         <translation>Color</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Show EOL characters</source>
-        <translation>Mostrar el caràcter de fi de línia</translation>
+        <translation>Mostra el caràcter de fi de línia</translation>
     </message>
     <message>
         <location line="+309"/>
         <source>Wrap long lines at current window border</source>
-        <translation>Ajusta les línies llargues a la vora de finestra actual</translation>
+        <translation>Ajusta les línies llargues a la vora de la finestra actual</translation>
     </message>
     <message>
         <location line="+475"/>
         <source>Default EOL mode</source>
         <translation>Identificador de fi de línia per defecte</translation>
     </message>
     <message>
         <location line="-420"/>
         <source>Indent width</source>
         <translation>Amplada del sagnat</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Tab indents line</source>
-        <translation>Sagna amb el tabulador</translation>
+        <translation>El tabulador sagna la línia</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Auto indentation</source>
         <translation>Sagnat automàtic</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Tab width</source>
-        <translation>Amplada del tabulador</translation>
+        <translation>Amplada del tabulat</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show indentation guides</source>
         <translation>Mostra els marcadors de sagnat</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -3613,42 +3587,41 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+120"/>
         <source>Match keywords</source>
         <translation>Coincidència de paraules clau</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Case sensitive</source>
-        <translation>Distingir entre majúscules i minúscules</translation>
+        <translation>Distingeix entre majúscules i minúscules</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Replace word by suggested one</source>
         <translation>Reemplaça la paraula pel suggeriment</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>Match words in document</source>
-        <translation>Coincidència de paraules al documento</translation>
+        <translation>Coincidència de paraules al document</translation>
     </message>
     <message>
         <location line="-540"/>
         <source>Use custom file editor</source>
         <translation>Utilitza un editor de fitxers personalitzat</translation>
     </message>
     <message>
         <location line="+868"/>
         <source>Editor Styles</source>
         <translation>Estils de l'editor</translation>
     </message>
     <message>
         <location line="+280"/>
         <source>Terminal Colors</source>
-        
         <translation>Colors del terminal</translation>
     </message>
     <message>
         <location line="-173"/>
         <location line="+854"/>
         <source>Font</source>
         <translation>Tipus de lletra</translation>
     </message>
@@ -3665,22 +3638,22 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="-137"/>
         <source>Show whitespace</source>
         <translation>Mostra els espais en blanc</translation>
     </message>
     <message>
         <location line="-45"/>
         <source>Do not show whitespace used for indentation</source>
-        <translation>No mostres els espais del sagnat</translation>
+        <translation>No mostres els espais de sagnat</translation>
     </message>
     <message>
         <location line="+863"/>
         <source>Number of characters before list is shown: </source>
-        <translation>Nombre de caràcters a partir de la qual es mostra la llista: </translation>
+        <translation>Nombre de caràcters a partir del qual es mostra la llista: </translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Create nonexistent files without prompting</source>
         <translation>Crea fitxers inexistents sense consultar</translation>
     </message>
     <message>
         <location line="-728"/>
@@ -3753,61 +3726,60 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="-135"/>
         <source>Normal</source>
         <translation>Normal</translation>
     </message>
     <message>
         <location line="+174"/>
         <source>Background inactive</source>
-        <translation>Fons inactivo</translation>
+        <translation>Fons inactiu</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>3D</source>
         <translation>3D</translation>
     </message>
     <message>
         <location line="+132"/>
         <source>Octave Startup</source>
         <translation>Inici de l'Octave</translation>
     </message>
     <message>
         <location line="+41"/>
         <location line="+1732"/>
         <source>Startup path</source>
-        <translation>Camí de l'inici</translation>
+        <translation>Camí d'inici</translation>
     </message>
     <message>
         <location line="-1719"/>
         <location line="+1712"/>
         <source>Browse</source>
         <translation>Explora</translation>
     </message>
     <message>
         <location line="-1635"/>
         <source>Tab width min.</source>
-        
-        <translation>Extensió mínima del tabulador.</translation>
+        <translation>Mida mínima de la pestanya.</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>max.</source>
         <translation>màx.</translation>
     </message>
     <message>
         <location line="+422"/>
         <location line="+140"/>
         <source>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</source>
         <translation>Funciona bé amb lletres d'espaiat fix. La línia es dibuixa en una posició basada en l'amplada del caràcter espai de la lletra per defecte. Pot no funcionar bé si l'estil utilitza fonts proporcionals o si es combinen lletres de mides diferents, o lletra normal amb negreta i/o cursiva.</translation>
     </message>
     <message>
         <location line="-463"/>
         <source>Enable Code Folding</source>
-        <translation>Activa el plegat del codi</translation>
+        <translation>Activa el plegament de codi</translation>
     </message>
     <message>
         <location line="+961"/>
         <source>Windows (CRLF)</source>
         <translation>Windows (CRLF)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -3847,32 +3819,31 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+65"/>
         <source>Comments (Octave)</source>
         <translation>Comentaris (Octave)</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Strings considered for uncommenting text</source>
-        <translation>Cadenes de text considerades per a text sense comentari</translation>
+        <translation>Cadenes de text a tenir en compte en descomentar</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>String used for commenting selected text</source>
-        <translation>Cadena de text utilitzada per comentar el text seleccionat</translation>
+        <translation>Cadena de text per comentar el text seleccionat</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Long lines</source>
         <translation>Línies llargues</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Break long lines at line length</source>
-        
         <translation>Divideix les línies llargues a partir de la longitud de línia</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Break lines only in comments</source>
         <translation>Només divideix les línies als comentaris</translation>
     </message>
     <message>
@@ -3888,33 +3859,32 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+10"/>
         <source>Line</source>
         <translation>Línia</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Background</source>
-        
         <translation>Segon pla</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Indentation</source>
         <translation>Sagnat</translation>
     </message>
     <message>
         <location line="+127"/>
         <source>Indentation uses tabs</source>
-        <translation>El sagnat utilitza el tabulador</translation>
+        <translation>El sagnat utilitza el tabulat</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Auto insert after "if" etc.</source>
-        <translation>Inserta automàticament després d'"if" etc.</translation>
+        <translation>Insereix automàticament després d'un "if" etc.</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Nothing</source>
         <translation>Res</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -3929,17 +3899,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+29"/>
         <source>Auto completion</source>
         <translation>Autocompleció</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>With Octave builtins</source>
-        <translation>Amb les funciones internes de l'Octave</translation>
+        <translation>Amb funcions internes de l'Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>With Octave functions</source>
         <translation>Amb funcions de l'Octave</translation>
     </message>
     <message>
         <location line="+64"/>
@@ -3954,17 +3924,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+19"/>
         <source>Reload externally changed files without prompt</source>
         <translation>Recarrega els fitxers modificats externament sense consultar</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Restore editor tabs from previous session on startup or when editor is shown again after closing</source>
-        <translation>Restaurar les pestanyes de l'editor de la sessió anterior a l'inici o quan l'editor es mostra de nou després d'haver-lo tancat</translation>
+        <translation>Restaura les pestanyes de l'editor de la sessió anterior</translation>
     </message>
     <message>
         <location line="+80"/>
         <source>Text encoding used for loading and saving</source>
         <translation>Codificació del text utilitzat per carregar i desar</translation>
     </message>
     <message>
         <location line="+25"/>
@@ -3984,17 +3954,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+28"/>
         <source>History buffer Size</source>
         <translation>Mida de l'historial</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Print debug location in terminal window in addition to the marker in the editor</source>
-        <translation>Imprimeix la ubicació de depuració a la Finestra del Terminal a més a més de a l'editor</translation>
+        <translation>Mostra la ubicació del depurador a la Finestra del Terminal, a més del marcador de l'editor</translation>
     </message>
     <message>
         <location line="+87"/>
         <source>Behavior</source>
         <translation>Comportament</translation>
     </message>
     <message>
         <location line="+30"/>
@@ -4055,17 +4025,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+17"/>
         <source>Network</source>
         <translation>Xarxa</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Allow Octave to connect to the Octave web site to display current news and information</source>
-        <translation>Permet a l'Octave la connexió al seu lloc web per mostrar novetats en quant a notícies i informació</translation>
+        <translation>Permet a l'Octave la connexió al lloc web per mostrar més informació i notícies actualitzades</translation>
     </message>
     <message>
         <location line="+46"/>
         <source>Use proxy server</source>
         <translation>Utilitza un servidor intermediari</translation>
     </message>
     <message>
         <location line="+90"/>
@@ -4115,17 +4085,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Alterna els colors de les files</translation>
     </message>
     <message>
         <location line="-290"/>
         <source>HttpProxy</source>
-        <translation>HttpProxy</translation>
+        <translation>Servidor intermediari HTTP</translation>
     </message>
     <message>
         <location line="-2179"/>
         <source>Icon set for dock widgets</source>
         <translation>Conjunt d'icones per als plafons d'acoblament</translation>
     </message>
     <message>
         <location line="-220"/>
@@ -4170,17 +4140,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
     <message>
         <location line="+90"/>
         <source>Action</source>
         <translation>Acció</translation>
     </message>
     <message>
         <location line="+83"/>
         <source>Socks5Proxy</source>
-        <translation>"Socks5Proxy"</translation>
+        <translation>Servidor intermediari SOCKS</translation>
     </message>
     <message>
         <location line="-16"/>
         <source>Hostname:</source>
         <translation>Nom de la màquina:</translation>
     </message>
     <message>
         <location line="+44"/>
@@ -4217,17 +4187,17 @@ Vols desar el conjunto de dreceres actual o cancel·lar l'acció?</translation>
         <location line="+4"/>
         <location line="+489"/>
         <source>System setting</source>
         <translation>Configuració de sistema</translation>
     </message>
     <message>
         <location line="-260"/>
         <source>IBeam Cursor</source>
-        <translation>Cursor en forma d'l</translation>
+        <translation>Cursor en forma d'I</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
         <translation>Cursor en forma de bloc</translation>
     </message>
     <message>
         <location line="+1"/>
